rs/nns/governance/canister/canister.rs

```diff
@@ -82,7 +83,9 @@ pub(crate) const LOG_PREFIX: &str = "[Governance] ";
 //
 // Do not access these global variables directly. Instead, use accessor
 // functions, which are defined immediately after.
-static mut GOVERNANCE: Option<Governance> = None;
+thread_local! {
+    static GOVERNANCE: RefCell<Option<Governance>> = RefCell::new(None);
```

Comment:
If we didn't need `governance_mut`, we could consider using `OnceCell<..>` instead of `RefCell<Option<..>>`.
In the `governance()` method, we could use `OnceCell::get` (no unsafe needed).
In the `set_governance()` method we could use `OnceCell::set`

Reply:
Thanks. I didn't know that. At least for this canister, we do need to be able to modify the global state.